access/src/main/webapp/WEB-INF/jsp/fullRecord/folderRecord.jsp

```diff
@@ -0,0 +1,69 @@
+<%--
+
+    Copyright 2008 The University of North Carolina at Chapel Hill
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+--%>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<%@ page trimDirectiveWhitespaces="true" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib prefix="cdr" uri="http://cdr.lib.unc.edu/cdrUI" %>
+
+<c:choose>
+    <c:when test="${not empty parentBriefObject.countMap}">
+        <c:set var="childCount" value="${parentBriefObject.countMap.child}"/>
+    </c:when>
+    <c:otherwise>
+        <c:set var="childCount" value="0"/>
+    </c:otherwise>
+</c:choose>
+
+<c:choose>
+    <c:when test="${childCount == 1}">
+        <c:set var="pluralCount" value="item"/>
+    </c:when>
+    <c:otherwise>
+        <c:set var="pluralCount" value="items"/>
+    </c:otherwise>
+</c:choose>
+<c:url var="fullRecordUrl" scope="page" value="record/${briefObject.id}"/>
+<div class="contentarea">
+    <div id="is-folder" class="columns browse-header">
+        <div class="column is-12">
+            <h2><c:out value="${parentBriefObject.title}"/> (<c:out value="${childCount}"/>)</h2>
```

Comment:
You can remove the child count display here. The mockups don't show it next to the title on the full record anymore, the number "(20529)" is actually the collection number and is part of the title.

You can actually do the same on the collection page too, only the admin unit appears to display the child count at the top.